<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>value_dialog</class>
 <widget class="QWidget" name="value_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ввод значения</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>351</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_log">
    <attribute name="title">
     <string>Логический</string>
    </attribute>
    <widget class="QRadioButton" name="radioButtonTrue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>True</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonFalse">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>False</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_num">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Количественный</string>
    </attribute>
    <widget class="QLineEdit" name="lineEditNum">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_num_poss">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Возможные значения</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_qual">
    <attribute name="title">
     <string>Качественный</string>
    </attribute>
    <widget class="QListWidget" name="listWidgetQual">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>211</width>
       <height>151</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Выберите значение из списка</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>label_5</zorder>
    <zorder>listWidgetQual</zorder>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите значение признака</string>
   </property>
  </widget>
  <zorder>pushButtonOK</zorder>
  <zorder>pushButtonBack</zorder>
  <zorder>label</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
